pkg/actors/actor_error.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package actors
+
+import (
+	"errors"
+	"fmt"
+
+	"google.golang.org/grpc/codes"
+
+	invokev1 "github.com/dapr/dapr/pkg/messaging/v1"
+)
+
```

Comment:
Consider adding a comment describing what this type is.

Also consider removing the `Actor` prefix from all type and func names since we have context that it is in the`actors` package. This also prevents stutter (`actors.ActorError` vs `actors.Error`). Even better to move this file into a new package `pkg/actors/errors` to separate concerns and prevent future circular imports.

Reply:
I would like to keep ActorError. I am moving to the new package and renaming other things.